zuul.d/layout.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# zuul.d/layout.yaml
+# This file is the Zuul layout of openstack/tripleo-ci project
+# It describes the jobs that will run when submitting a change in this
+# repository.
+
+- project:
+    check:
+      jobs: &defaults
+        - ansible-tox-linters:
+            vars:
+              tox_envlist: lint
+        - ansible-tox-py27
+        # Due: https://github.com/ansible/ansible-zuul-jobs/issues/292
```

Comment:
Fixed, just waiting for code to merge: https://github.com/ansible/ansible-zuul-jobs/pull/293